crates/rooch-rpc-client/src/lib.rs

```diff
@@ -169,6 +181,125 @@ impl Client {
     }
 }
 
+#[async_trait]
+impl<M, S> EthAPI for Client<M, S>
```

Comment:
This implementation should be auto-generated by the RPC framework, do not manually implement.

Reply:
Will verify the generation logics later.